explore/circadian-efficacy/data_id_mapping.ipynb

```diff
@@ -0,0 +1,987 @@
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>drugbank_id</th>\n",
+       "      <th>name</th>\n",
+       "      <th>type</th>\n",
+       "      <th>groups</th>\n",
+       "      <th>atc_codes</th>\n",
+       "      <th>categories</th>\n",
+       "      <th>inchikey</th>\n",
+       "      <th>inchi</th>\n",
+       "      <th>description</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>DB00001</td>\n",
+       "      <td>Lepirudin</td>\n",
+       "      <td>biotech</td>\n",
+       "      <td>approved</td>\n",
+       "      <td>B01AE02</td>\n",
+       "      <td>Antithrombins|Fibrinolytic Agents</td>\n",
+       "      <td>NaN</td>\n",
+       "      <td>NaN</td>\n",
+       "      <td>Lepirudin is identical to natural hirudin exce...</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>DB00002</td>\n",
+       "      <td>Cetuximab</td>\n",
+       "      <td>biotech</td>\n",
+       "      <td>approved</td>\n",
+       "      <td>L01XC06</td>\n",
+       "      <td>Antineoplastic Agents</td>\n",
+       "      <td>NaN</td>\n",
+       "      <td>NaN</td>\n",
+       "      <td>Epidermal growth factor receptor binding FAB. ...</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "  drugbank_id       name     type    groups atc_codes  \\\n",
+       "0     DB00001  Lepirudin  biotech  approved   B01AE02   \n",
+       "1     DB00002  Cetuximab  biotech  approved   L01XC06   \n",
+       "\n",
+       "                          categories inchikey inchi  \\\n",
+       "0  Antithrombins|Fibrinolytic Agents      NaN   NaN   \n",
+       "1              Antineoplastic Agents      NaN   NaN   \n",
+       "\n",
+       "                                         description  \n",
+       "0  Lepirudin is identical to natural hirudin exce...  \n",
+       "1  Epidermal growth factor receptor binding FAB. ...  "
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# read in Drugbank ID map\n",
+    "drugbank = pd.read_csv('https://raw.githubusercontent.com/yhao-compbio/drugbank/gh-pages/data/drugbank.tsv', sep = '\\t', header = 0)\n",
```

Comment:
Change URL to:

https://github.com/dhimmel/drugbank/raw/6b9ae386d6ba4a0eca2d66d4b0337a6e90fe81f4/data/drugbank.tsv

Note the use of commit hash versioning (press `y` hotkey on github). Also see use of `raw` which you can switch to `blob` to get back to HTML view.

Switch any other links to GitHub-hosted datasets in this PR to use this format.